features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/media/local/LocalMediaView.kt

```diff
@@ -186,3 +212,50 @@ fun MediaPDFView(
     }
     PdfViewer(pdfViewerState = pdfViewerState, modifier = modifier)
 }
+
+@Composable
+fun MediaFileView(
+    uri: Uri?,
+    info: MediaInfo?,
+    onReady: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    LaunchedEffect(Unit) {
+        if(uri != null) {
+            onReady()
+        }
+    }
+    Box(modifier = modifier, contentAlignment = Alignment.Center) {
+        Column(horizontalAlignment = Alignment.CenterHorizontally) {
+            Box(
+                modifier = Modifier
+                    .size(72.dp)
+                    .clip(CircleShape)
+                    .background(MaterialTheme.colorScheme.onBackground),
+                contentAlignment = Alignment.Center,
+            ) {
+                Icon(
+                    imageVector = Icons.Outlined.Attachment,
+                    contentDescription = "OpenFile",
```

Comment:
Should be a string resource so it gets translated

Reply:
I think there is no need for contentDescription here as it doesn't convey anything important, I'll remove it